src/wix/WixToolset.Core/ExtensibilityServices/WixBranding.cs

```diff
@@ -10,6 +10,7 @@ namespace WixToolset.Core.ExtensibilityServices
     using System.Diagnostics;
     using System.IO;
     using System.Reflection;
+
```

Comment:
Why add whitespace here?

Reply:
That was a result of VS doing a 'remove unused usings and sort'. It's gone.